CHANGELOG.md

```diff
@@ -5,6 +5,11 @@
 ### Fixes
 
 - Fix high percentage of slow frames (#1915)
+## Unreleased
+
+### Features
+
+- Upload frame rendering timestamps to correlate to sampled backtraces (#1910)
```

Comment:
This needs to move up to a new `unreleased` section.

Reply:
Lovely. Bad automatic rebase merge. Fixed now.